helpers/labgrid-bound-connect

```diff
@@ -34,6 +35,9 @@ def main(ifname, address, port):
     else:
         raise RuntimeError("Invalid IP version '{}'".format(address.version))
 
+    # Delete the IP lookup cache for the address in case it is stale
+    subprocess.run(["ip", "neigh", "del", str(address), "dev", ifname])
```

Comment:
We need to make sure that no output is produced on stdout, as that would be indistinguishable from data sent be the remote side.